source/_docs/getting-support.md

```diff
@@ -6,8 +6,9 @@ categories: [managing]
 tags: [platform]
 ---
 Submit a ticket with our Support Team to request assistance on issues within our [scope of support](/docs/scope-of-support/).
+
 ## File a Ticket Through the Dashboard
-1. Create a support ticket from within the site Dashboard by clicking **Support > Open Ticket**. This will automatically include the site and user information.
+1. Create a support ticket from within the site's Dashboard by clicking **Support > Open Ticket**. This will automatically include the site and user information.

```

Comment:
Tickets can be created through three gateways: the user gateway, the site gateway, and the organization gateway. A user can only file site-specific support tickets from the site gateway in the manner you describe. A user submit a site-specific ticket from the organization's gateway but additionally choose an option that the ticket does not pertain to a site. Finally, the user can submit a site-specific ticket from their user gateway or submit a ticket that does not pertain to a site. The user's membership to sites and organizations apply to all three gateways.